docs/source/zh/_toctree.yml

```diff
@@ -68,8 +68,10 @@
   title: 概念指南
 - sections:
   - sections:
+    - local: main_classes/configuration
+      title: configuration
```

Comment:
Shouldn't we align the title ("configuration") here with the title of the doc ("配置")? Also curious why you chose to revert "模型" to just "model" :)

Reply:
I'm not sure whether it's necessary to translate concepts like 'model' and 'configuration' into Chinese because, in the field of deep learning, people with any language background can understand them intuitively :)
However, to maintain consistency throughout the document, I have now re-translated them into their respective Chinese terms :)